doc/installation.rst

```diff
@@ -61,7 +61,7 @@ Supported Versions
 
 
 SmartSim supports multiple machine learning libraries through
-the use of RedisAI_. The following libraries are supported.
+the use of RedisAI_ 1.2.7. The following libraries are supported (based on .
```

Comment:
Looks like you didn't finish a thought here

Reply:
Thank you fixed.